pkg/plugin/plugin.go

```diff
@@ -82,6 +92,8 @@ func (p *plugin) ValidateAdmin(ctx context.Context, new, old *api.OpenShiftManag
 	validator := validate.NewAdminValidator(p.testConfig.RunningUnderTest)
 	errs = validator.Validate(new, old)
 
+	// TODO figure out what kind of validation we want here for nameservers.
+
```

Comment:
@jim-minter remember this..

Reply:
I don't think there's any validation to be done here?